frontend/src/metabase/forms/components/FormSubmitButton/FormSubmitButton.tsx

```diff
@@ -37,6 +39,12 @@ export const FormSubmitButton = forwardRef(function FormSubmitButton(
   });
   const submitColor = getSubmitButtonColor(status, { color });
 
+  useEffect(() => {
```

Comment:
this triggers the `success` state that allows us to close the modal when the question is saved

Reply:
yeah, fair enough. that's a good call

Reply:
fixed. I moved the logic to the provider and the modal component itself

Reply:
Did you forget to push?

Reply:
Yes I did 🙃